server/ldap_check.rb

```diff
@@ -0,0 +1,107 @@
+# Operations with LDAP.  Currently only supports getting default configuration values
+# and checking if a user is in a particular mcommunity group.
+# A configuration file name can be passed in when creating the object.  The default
+# value is ./server/local/ldap.yml.
+
+require 'net-ldap'
+require 'yaml'
+require_relative '../server/Logging'
+
+class LdapCheck
+  include Logging
+
+  ## Setup the ldap connection object and save the members of the group.
+  def initialize(args={})
```

Comment:
Does this mean we need to restart the Student Dashboard in order to add new members to the group? I think this is ok, but it should be documented - I'd suggest a comment here and also in the yml file where the group is defined